client/src/App.jsx

```diff
@@ -1,32 +1,38 @@
-import './styles/App.css';
+import "./styles/App.css";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
-import { Typography } from '@mui/material';
-import Nav from './Nav.jsx';
-import TakeQuiz from './components/take-quiz/TakeQuiz.jsx';
-import Dashboard from './components/user-dashboard/Dashboard.jsx';
-import Login from '../src/pages/Login';
-import Register from '../src/pages/Register';
+// import { BrowserRouter as Router, Route, Routes, Link } from "react-router-dom";
+import { Typography } from "@mui/material";
+import Nav from "./Nav.jsx";
+import TakeQuiz from "./components/take-quiz/TakeQuiz.jsx";
+import Dashboard from "./components/user-dashboard/Dashboard.jsx";
+import Login from "../src/pages/Login";
+import Register from "../src/pages/Register";
 
 function App() {
   return (
     <Router>
       <Nav />
       <Routes>
-          <Route path='/*' element={
-            <div className='body'>
-                <Typography variant="h4">Welcome to Quiz Whiz</Typography>
+        <Route
+          path="/*"
+          element={
+            <div className="body">
+              <Typography variant="h4">Welcome to Quiz Whiz</Typography>
+              {/* Using this for Take Quiz Development. Please leave here for now  */}
```

Comment:
I think including the Development Links in the PR would be helpful

Reply:
Removing these entirely, Thanh patched the Nav issue in a cleaner way and these aren't necessary anymore